### PR TITLE
doc: add document for HTTP header

### PR DESCRIPTION
**Pull request by qingbozhang**
This PR for #4901

### CODE REVIEW

File: config/confighttp/README.md [hunk 1]
```diff
@@ -71,6 +71,9 @@ will not be enabled.
 [cors-headers]: https://developer.mozilla.org/en-US/docs/Glossary/CORS-safelisted_request_header
 [cors-cache]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age
 [origin]: https://developer.mozilla.org/en-US/docs/Glossary/Origin
+[attribute-processor]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/attributesprocessor/README.md
+
+you can enable [`attribute processor`][attribute-processor] to append `X-Forwarded-For` value to span's attribute using custom key.
```
Comment (jpkrohling) [inline config/confighttp/README.md:76]: Any HTTP header can be added, not only this one.

File: config/confighttp/README.md [hunk 2]
```diff
@@ -87,4 +90,10 @@ receivers:
             - Example-Header
           max_age: 7200
         endpoint: 0.0.0.0:55690
-```
+processors:
```
Comment (jpkrohling) [inline config/confighttp/README.md:93]: You also need to enable the "include_metadata", no?

Comment (qingbozhang) [reply config/confighttp/README.md:93]: Sorry, I forget it.  need to enable the "include_metadata"

File: config/confighttp/README.md (hunk 1 above)
Comment (qingbozhang) [reply config/confighttp/README.md:76]: yes.  I'll fix it

**Comment by jpkrohling** [comment]
What's the status of this PR?

**Review by codeboten — COMMENTED** [review]
@qingbozhang please sign the CLA and we can move ahead with the PR

**Comment by qingbozhang** [comment]
I have signed CLA.  I don't know why still missing CLA Authorization.

**Comment by jpkrohling** [comment]
This seems to be the problem:

> x The commit (https://github.com/open-telemetry/opentelemetry-collector/commit/282998986fcc659598021679090284bc3b784bb9). This user is missing the User's ID, preventing the EasyCLA check. [Consult GitHub Help](https://help.github.com/en/github/committing-changes-to-your-project/why-are-my-commits-linked-to-the-wrong-user) to resolve.For further assistance with EasyCLA, [please submit a support request ticket](https://jira.linuxfoundation.org/servicedesk/customer/portal/4).

**Comment by qingbozhang** [comment]
Maybe I  need to reopen a PR.

**Pull request closed without merging**